packages/amplify_core/lib/src/util/get_base_element_href_from_dom.web.dart

```diff
@@ -0,0 +1,9 @@
+import 'dart:html';
+
+import 'package:amplify_core/amplify_core.dart';
+
+/// {@macro amplify_core.get_base_element_href_from_dom}
+String? getBaseElementHrefFromDom() {
+  if (!zIsFlutter) return null;
```

Comment:
Yeah, it was easiest just to not even consider non-flutter apps for now, but I can add some logic to include them.

Flutter apps always include the `href` attribute and it always excludes the origin. In a non-flutter app, this might include the origin. It might not have a `href`. There might be other cases that I am unaware of. So I took the easy way out and just returned null since we don't really support non-flutter use cases at the moment.

I'll update it to at least hand the cases I am aware of.

Reply:
I updated this just to remove the `zIsFlutter` check. We can update this as needed if we support non-flutter use cases.